packages/@angular/cli/models/webpack-configs/common.ts

```diff
@@ -67,12 +67,12 @@ export function getCommonConfig(wco: WebpackConfigOptions) {
     devtool: buildOptions.sourcemaps ? 'source-map' : false,
     resolve: {
       extensions: ['.ts', '.js'],
-      modules: ['node_modules', nodeModules],
+      modules: [nodeModules, 'node_modules']
```

Comment:
Keep this one unchanged.  Standard node module resolution needs to take precedence to avoid incorrect transitive package versions from being resolved.